app/pulp/app/models/auth.py

```diff
@@ -0,0 +1,63 @@
+"""
+Django models related to the Authentication system
```

Comment:
There are things in here that don't seem like they're doing things in obvious ways, at least to me. It'd be helpful to have a link to https://docs.djangoproject.com/en/1.8/topics/auth/customizing/#specifying-a-custom-user-model in here somewhere to make it clear that that's what's going on in here, and that future developers should probably be familiar with that page before changing things around in here.